### ISSUE TITLE
ebuilds QA

### ISSUE THREAD

**Issue by Alessandro-Barbieri** [ros/ros-overlay#34]
Don't `wget` in `src_unpack`

**Comment by allenh1** [comment]
I would prefer not to as well. It's a rather hacjy fix for a thing I have not yet figured out how to do with portage.

**Comment by Alessandro-Barbieri** [comment]
Is portage downloading the correct files?

**Comment by allenh1** [comment]
No, no it is not. I think I know how to fix this, and will try when I get into the office this morning.

**Comment by allenh1** [comment]
Let me rephrase: it downloads the right source sometimes. It gets confused by the version numbers...

If your package is version 0.6.3, and another package is if that version, then portage caches the first one. Then, when emerging the second, you re-emerge the first (since it used the cached version).

**Comment by Alessandro-Barbieri** [comment]
Use the "->" operator to rename what you download

**Comment by allenh1** [comment]
Got it. I'll fix that as soon as I get in.
